src/04-markup/04-parsing_02.md

```diff
@@ -512,10 +512,9 @@ trim = unwords . words
 
 In an earlier chapter, we parsed a few examples of our markup language [by hand](01-data_type.html#exercises).
 Now, we can try to test our parser by comparing our solutions to our parser.
-By adding `Eq` to the list of type class instances to derive for our `Structure` data type
-(as shown on the line marked with (1) in "final module" above),
-we can add these to our module and use the `==` (equals) operator to compare our
-solutions to the result our parser gives.
+By deriving `Eq` for our `Structure` data type
+(marked with (1) in "final module" above),
+we can compare solutions with the `==` (equals) operator.
```

Comment:
The difference from the old version and the new version here is that the new version says *what* we do, and the old version says *how* to do it. I think at this point it is important to say how to do it, but I agree that the phrasing isn't great.

Reply:
I find the 'how ' is clear from code snippet (1). Also you have explained `show` in detail in the previous section already.